CHANGELOG.md

```diff
@@ -1,7 +1,26 @@
-## next release
+## 2.3.0 / 2018-06-05
 
+* [FEATURE] Add query commands to promtool
+* [FEATURE] Add security headers to HTTP server responses
+* [FEATURE] Pass query hints via remote read API
 * [CHANGE] `marathon_sd`: use `auth_token` and `auth_token_file` for token-based authentication instead of `bearer_token` and `bearer_token_file` respectively.
```

Comment:
The order is supposed to be change, feature, enhancement, bugfix https://github.com/prometheus/prometheus/wiki/HOWTO-cut-a-new-release